src/main/java/uk/gov/companieshouse/certifiedcopies/orders/api/interceptor/LoggingInterceptor.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.certifiedcopies.orders.api.interceptor;
+
+import org.springframework.lang.Nullable;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.ModelAndView;
+import org.springframework.web.servlet.handler.HandlerInterceptorAdapter;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+import uk.gov.companieshouse.logging.util.RequestLogger;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import static uk.gov.companieshouse.certifiedcopies.orders.api.logging.LoggingUtils.APPLICATION_NAMESPACE;
+
+@Component
+public class LoggingInterceptor extends HandlerInterceptorAdapter implements RequestLogger {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(APPLICATION_NAMESPACE);
```

Comment:
On the item-handler service, Vaughan made a good point that we can keep the namespace in the LoggingUtils class and get the Logger from the class? It seems logical to do the same in this service?
Could the application namespace reference be removed and the Logger be made available from the LoggingUtils class instead?